tests/api/fuzzer.py

```diff
@@ -0,0 +1,319 @@
+import argparse
+import json
+import logging
+import random
+import re
+import requests
+import sys
+import posixpath
+import urllib.parse
+
+
+#######################################################################################################################
+### Utilities
+
+def some(s):
+    return random.choice(sorted(s))
+
+def pretty_json(tree, depth=0, max_depth=None):
```

Comment:
Why not just use `json.dumps()`?

Reply:
It's really just for readability - the inlined version of the swagger spec is several pages long and that method just splits elements onto lines and indents the tree to show the shape.

Reply:
`dumps()` is prettifying input.